controller/src/beerocks/master/son_management.cpp

```diff
@@ -814,6 +814,14 @@ void son_management::handle_cli_message(Socket *sd,
         message_com::send_cmdu(sd, cmdu_tx);
         break;
     }
+    
+    //=======================
+    case beerocks_message::ACTION_CLI_OREN_PRINT: {
+        LOG(TRACE) << "========> son_management.cpp[820] beerocks_message::ACTION_CLI_OREN_PRINT";
```

Comment:
no need to print file/line , its part of the logs out put anyway

Reply:
fixed